pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -977,7 +978,7 @@ def retrieve(self, request, *args, **kwargs):
         # the contents of a repo version can be cached without worry because repo
         # versions are immutable
         cache_key = (
-            f"version-details-{repo_version.pk}"
+            f"{settings.ANSIBLE_API_HOSTNAME}-version-details-{repo_version.pk}"
```

Comment:
Should there be a few "-" to separate the repository version from the namespace and the name from the namespace?

e.g. "ansible-spear-1.0.0" should have a different key than "ansibles-pear-1.0.0".

Reply:
good point